From: Kay Sievers <kay@vrfy.org>
Date: Mon, 26 May 2014 09:30:21 +0800
Subject: udev: keyboard - also hook into "change" events

Re-apply the keymaps when "udevadm trigger" is called. Hooking into
"add" only would just remove all keymap content from the udev database
instead of applying the new config.

(cherry picked from commit 49804365ea1242456c9763058a59cf68479e07ea)
(cherry picked from commit 8d9518eb8f0dbf20e0abfb3b97845ff1f22d8dc3)
---
 rules/60-keyboard.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rules/60-keyboard.rules b/rules/60-keyboard.rules
index 4e0f366..22f71e7 100644
--- a/rules/60-keyboard.rules
+++ b/rules/60-keyboard.rules
@@ -1,6 +1,6 @@
 # do not edit this file, it will be overwritten on update
 
-ACTION!="add", GOTO="keyboard_end"
+ACTION=="remove", GOTO="keyboard_end"
 KERNEL!="event*", GOTO="keyboard_end"
 ENV{ID_INPUT_KEY}=="", GOTO="keyboard_end"
 
